config/boards/odroidn2.conf

```diff
@@ -24,6 +23,11 @@ write_uboot_platform_mtd() {
 	dd if=$1/u-boot-spi.bin of=/dev/mtdblock0
 }
 
+function post_config_uboot_target__extra_configs_for_odroid_n2() {
```

Comment:
I was enabling btrfs support in u-boot for the Helios4 yesterday (I used straight patching) but this is of course more elegant.  I suggest to carve this out into an extension so it can be used more easily.

PR incoming.